kibble/render/render.go

```diff
@@ -69,7 +69,8 @@ func Watch(sourcePath string, buildPath string, cfg *models.Config, port int32,
 
 // Render - render the files
 func Render(sourcePath string, buildPath string, cfg *models.Config) int {
-
+	
+	fmt.Println("remoteTranslations is: ",cfg.RemoteTranslations)
```

Comment:
I wonder if `log.Infof("remoteTranslations: %t", cfg.RemoteTranslations)` would be better for consistency.